CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 cmake_minimum_required (VERSION 3.5)
-cmake_policy(SET CMP0074 NEW) # Don't complain about using BOOST_ROOT...
+if(CMAKE_VERSION VERSION_EQUAL 3.12 OR CMAKE_VERSION VERSION_GREATER 3.12)
```

Comment:
`NOT LESS` might be a bit simpler? I assume we are not using `GREATER_EQUAL` because it was introduced in cmake later than 3.5?

Reply:
Yes it is a feature added in cmake 3.7.